scripts/import-npa/natural_protected_areas.sh

```
@@ -0,0 +1,461 @@
+prepare_temp_dir() {
+  print_message info "Creating temporary directory..."
+  mkdir -p "$TEMP_DIR"
+  mkdir -p "$EXTRACT_DIR"
+  print_message success "Created temporary directory: $TEMP_DIR"
+}
+
+# Download the dataset
+download_dataset() {
+  print_message info "Preparing to download dataset from $DATASET_URL..."
+  
+  local zip_name="protected_planet_$(date +%Y%m%d%H%M%S).zip"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)